CONTRIBUTING.md

```diff
@@ -23,3 +23,28 @@ If you'd like to help make RSpec better, here are some ways you can contribute:
 If you need help getting started, check out the [DEVELOPMENT](DEVELOPMENT.md) file for steps that will get you up and running.
 
 Thanks for helping us make RSpec better!
+
+## `Small` issues
+
+Small issues are the ones that we be believe are best suited for new
+contributors to get started on. They represent a meaningful contribution to the
+project that can be made that should not be too hard to pull off.
+
+## `Triage` issues
+
+Triage issues are ones that have been labelled by the maintainers that we
+believe do not currently have enough information for them to be reproduced by
+us. You can help us move triage issues into `ready to go` or `has reproduction
+case` (for feature requests or bugs) as a seriously meaningful contribution.
+
+## `Has reproduction case` issues
+
+Issues that have reproduction cases have a repository that we can clone that
+enable us to quickly determine the issue is valid and then start debugging
+within RSpec. These issues are good ones to tackle to help us actively fix bugs.
```

Comment:
The interactions between rspec and rails in rspec-rails and subtle enough that having a clonable repo is pretty important here.  For the other repos, it's much less so; often times the description from the user is sufficient, or a snippet from them.  Sometimes we can't reproduce and do ask for a repo.